contribs/gmf/src/search/component.js

```diff
@@ -868,7 +881,7 @@ gmf.SearchController = class {
    * @param {jQuery.Event} event Event.
    * @param {Object|ol.Feature} suggestion Suggestion.
    * @param {TypeaheadDataset} dataset Dataset.
-   * @this {gmf.SearchController}
+   * @this {gmf.search.component.SearchController_}
```

Comment:
Have you tried removing the `@this` annotation and the `static`? Would be good to fix this if that does not take too much time.